ci-operator/step-registry/openshift/e2e/openstack/ccm/day1/openshift-e2e-openstack-ccm-day1-workflow.yaml

```diff
@@ -0,0 +1,21 @@
+workflow:
+  as: openshift-e2e-openstack-ccm
+  steps:
+    pre:
+    - chain: ipi-openstack-pre-noramfs
+    - ref: ccm-conf-apply-feature-gate
+    test:
+      - ref: openshift-e2e-test
+    post:
+    - chain: ipi-openstack-post
+    env:
+      CONFIG_TYPE: "minimal"
+      BASE_DOMAIN: shiftstack.devcluster.openshift.com
+      TEST_SKIPS: >-
+        Managed cluster should start all core operators\|
+        Alerts shouldn't report any alerts in firing or pending state\|
+        Prometheus when installed on the cluster shouldn't report any alerts in firing state apart from Watchdog and AlertmanagerReceiversNotConfigured
```

Comment:
Are these skips due to the lack of CSI migration?

Reply:
right; let's remove the skips until we know they're useful.